### ISSUE TITLE
Commands not working when using MediaResponse API

### ISSUE THREAD

**Issue by joecaine** [actions-on-google/actions-on-google-nodejs#310]
I'm using the MediaResponse API to serve audio files in my responses. Users are prompted to say 'next' or 'back' to my action, after which I serve up another audio file. As per the GA documentation, this should work no problem. Indeed it works fine in the simulator. However, I get no request into my action when the user says these words on an actual device. Even workaround words like 'forward' are now causing the action to crash.

I've noticed that the TED action https://assistant.google.com/services/a/uid/000000fc24534964 uses similar functionality to the one I'm trying to implement, and theirs isn't working either.

Every time I work around an issue, Google introduces a new platform bug that breaks my fix. Who is testing this? Is there any chance of this API ever working properly?

**Comment by igilham** [comment]
Same issue observed, but note that "next" currently seems to work on the Google Home, but not on Android.

There is very little consistency between devices and something different seems to be broken every week.

**Comment by duyleomessi** [comment]
Did you solve the problem? I also met the same problem.

**Comment by igilham** [comment]
Not really. The service is very flaky. Occasionally a use case gets fixed but they often break again later with no communication from Google.

We've seen this issue with several different variations on "next", "previous", "stop" etc. in a couple different languages.
